### ISSUE TITLE
Incompatibility with carpet-fixes

### ISSUE THREAD

**Issue by igoticecream** [fxmorin/WeirdAddons#6]
I'm using `carpet-fixes@1.17.1-1.1.0` and `weirdaddons@1.17-1.4.0` and this is the error log:

```
[18:21:13] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/Pedro/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.9.4+mixin.0.8.2/sponge-mixin-0.9.4+mixin.0.8.2.jar Service=Knot/Fabric Env=CLIENT
[18:21:13] [main/INFO]: Compatibility level set to JAVA_16
[18:21:13] [main/WARN]: Reference map 'mixins.nonmmlauncher.replaymod.refmap.json' for mixins.nonmmlauncher.replaymod.json could not be read. If this is a development environment you can ignore this message
[18:21:13] [main/WARN]: @Mixin target net/minecraft/class_2474$class_5124 is public in fabric-tag-extensions-v0.mixins.json:MixinObjectBuilder and should be specified in value
[18:21:14] [main/WARN]: Error loading class: shadersmod/client/ShadersRender (java.lang.ClassNotFoundException: shadersmod/client/ShadersRender)
[18:21:14] [main/WARN]: Error loading class: net/coderbot/iris/uniforms/CommonUniforms (java.lang.ClassNotFoundException: net/coderbot/iris/uniforms/CommonUniforms)
[18:21:14] [main/WARN]: Error loading class: net/coderbot/iris/Iris (java.lang.ClassNotFoundException: net/coderbot/iris/Iris)
[18:21:15] [main/INFO]: [OptiFine] (Reflector) Field not found: World.tileEntitiesToBeRemoved
[18:21:15] [main/WARN]: Method overwrite conflict for method_5800 in carpet.mixins.json:ItemEntityMixin, previously written by carpetfixes.mixins.itemFixes.ItemEntity_lightningKillsDropsMixin. Skipping method.
[18:21:16] [main/FATAL]: Mixin apply failed carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> net.minecraft.class_2502: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Specified index 2 for @ModifyArg is invalid for args (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;I), expected I on net/minecraft/class_2502::spongeUpdate [INJECT Applicator Phase -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> Apply Injections ->  -> Inject -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin->@ModifyArg::spongeUpdate(I)I]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Specified index 2 for @ModifyArg is invalid for args (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;I), expected I on net/minecraft/class_2502::spongeUpdate [INJECT Applicator Phase -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> Apply Injections ->  -> Inject -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin->@ModifyArg::spongeUpdate(I)I]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.findArgIndex(ModifyArgInjector.java:157) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.injectAtInvoke(ModifyArgInjector.java:112) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.InvokeInjector.inject(InvokeInjector.java:82) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.inject(ModifyArgInjector.java:102) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:264) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:385) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1284) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) ~[fabric-loader-0.11.7.jar:0.9.4+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:162) ~[fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:154) ~[fabric-loader-0.11.7.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	at net.minecraft.class_3523.<clinit>(class_3523.java:13) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5471.<clinit>(class_5471.java:11) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5458.method_30573(class_5458.java:44) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:75) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:723) [?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:74) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:266) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:44) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:139) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:567) ~[?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.7.jar:?]
```

Removing any, fixes or weirdaddons, solve the crash.

**Comment by FxMorin** [comment]
Could you check if the bug is fixed in [this](https://github.com/fxmorin/WeirdAddons/suites/3925373789/artifacts/98216843)

**Comment by igoticecream** [comment]
Same issue i believe, im using the artifact linked in the above comment

```
[17:36:29] [main/INFO]: Trying to switch memory allocators to work around memory leaks present with Jemalloc 5.0.0 through 5.2.0 on Windows
[17:36:29] [main/INFO]: I used the json to destroy the json
[17:36:29] [main/INFO]: Smooth Boot config initialized
[17:36:30] [main/FATAL]: Mixin apply failed carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> net.minecraft.class_2502: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Specified index 2 for @ModifyArg is invalid for args (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;I), expected I on net/minecraft/class_2502::spongeUpdate [INJECT Applicator Phase -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> Apply Injections ->  -> Inject -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin->@ModifyArg::spongeUpdate(I)I]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Specified index 2 for @ModifyArg is invalid for args (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;I), expected I on net/minecraft/class_2502::spongeUpdate [INJECT Applicator Phase -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> Apply Injections ->  -> Inject -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin->@ModifyArg::spongeUpdate(I)I]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.findArgIndex(ModifyArgInjector.java:157) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.injectAtInvoke(ModifyArgInjector.java:112) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.InvokeInjector.inject(InvokeInjector.java:82) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.inject(ModifyArgInjector.java:102) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:264) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:385) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1284) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) ~[fabric-loader-0.11.7.jar:0.9.4+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:162) ~[fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:154) ~[fabric-loader-0.11.7.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	at net.minecraft.class_3523.<clinit>(class_3523.java:13) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5471.<clinit>(class_5471.java:11) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5458.method_30573(class_5458.java:44) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:75) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:723) [?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:74) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:266) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:44) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:139) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:567) ~[?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.7.jar:?]
```

**Comment by FxMorin** [comment]
Alright I believe that needs to be fixed in carpet-fixes. One moment

**Comment by FxMorin** [comment]
Can you try replacing carpet-fixes with: [fix](https://github.com/fxmorin/carpet-fixes/suites/3932663874/artifacts/98480834)

**Comment by igoticecream** [comment]
This is the error I get when using the two artifacts you have linked together:

```
[14:57:32] [main/FATAL]: Mixin apply failed carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> net.minecraft.class_2502: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Specified index 2 for @ModifyArg is invalid for args (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;I), expected I on net/minecraft/class_2502::spongeUpdate [INJECT Applicator Phase -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> Apply Injections ->  -> Inject -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin->@ModifyArg::spongeUpdate(I)I]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Specified index 2 for @ModifyArg is invalid for args (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;I), expected I on net/minecraft/class_2502::spongeUpdate [INJECT Applicator Phase -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin -> Apply Injections ->  -> Inject -> carpet-fixes.mixins.json:blockUpdates.SpongeBlock_missingUpdateMixin->@ModifyArg::spongeUpdate(I)I]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.findArgIndex(ModifyArgInjector.java:157) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.injectAtInvoke(ModifyArgInjector.java:112) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.InvokeInjector.inject(InvokeInjector.java:82) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.inject(ModifyArgInjector.java:102) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:264) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:385) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1284) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) ~[sponge-mixin-0.9.4+mixin.0.8.2.jar:0.9.4+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) ~[fabric-loader-0.11.7.jar:0.9.4+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:162) ~[fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:154) ~[fabric-loader-0.11.7.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	at net.minecraft.class_3523.<clinit>(class_3523.java:13) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5471.<clinit>(class_5471.java:11) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5458.method_30573(class_5458.java:44) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:75) ~[intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:723) [?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:74) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:266) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:44) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:139) [intermediary-fabric-loader-0.11.7-1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:567) ~[?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153) [fabric-loader-0.11.7.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.7.jar:?]
```

**Comment by igoticecream** [comment]
I created a pull request with a solution https://github.com/fxmorin/carpet-fixes/pull/20

**Comment by FxMorin** [comment]
Accepted
